about.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Georgia Adam</title>
+  <link rel="shortcut icon" href="/assets/forest.png">
+  <link rel="stylesheet" href="/styles/style.css">
+  <link rel="stylesheet" href="/styles/about.css">
+
+</head>
+
+<body>
+  <nav>
+    <ul>
+      <li><a href="/index.html"> Home </a></li>
+      <li><a href="/portfolio.html"> Portfolio </a></li>
+      <li><a href="/about.html"> About </a></li>
+    </ul>
+  </nav>
+  
+  <header>
+    <h1>Georgia Adam</h1>
+    <h2>Software Engineering Student @ Ada Developers Academy</h2>
+  </header>
+
+  <section id="about_me">
+    <h3>
+      About me.
+    </h3>
+  </section>
+
+  <main class="grid_container">
+    <div id="paw"><img alt="Pawprint" src="assets/track.png"/></div>
+    <div id="matcha"><img alt="Matcha" src="assets/matcha.png"/></div>
+    <h3 id="bio">Dog lover. Matcha drinker.</h3>
+  </main>
+
+  <section class="contact">
+    <h3>Contact</h3>
+    <ul>
+      <li><a href="mailto:georgiaeadam@gmail.com"><img alt="Email" src="assets/square.png"/></a></li>
+      <li><a href="https://www.linkedin.com/in/georgiaeadam/"><img alt="LinkedIn" src="assets/linkedin.png"/></a></li>
+    </ul>
+  </section>
+
+  <footer>
+    <h4>&copy; 2022 Georgia Adam <br>
+    Photo by <a href="https://unsplash.com/@aewild?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Amanda Frank</a> on <a href="https://unsplash.com/@aewild?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a> / 
```

Comment:
Since this line is long and there's another set of anchor tags that you might need to scroll to see, I would recommend breaking this into 2 lines so it's harder to miss content when skimming down the HTML.
```suggestion
    Photo by <a href="https://unsplash.com/@aewild?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Amanda Frank</a> /
    on <a href="https://unsplash.com/@aewild?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a> / 
```